ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -128,6 +130,7 @@ public void run() {
       initializationException.compareAndSet(null, e);
     }
     if (initializationException.get() == null) {
+      sealedStateChangeCounter.incrementAndGet();
```

Comment:
ClusterChangeHandler will not increment this during initial calls, only on subsequent calls. It could be written differently, but this seems harmless and only increments once during initialization, and once is sufficient.